ios/RNGoogleMobileAds/RNGoogleMobileAdsConsentModule.m

```diff
@@ -165,7 +162,7 @@ - (NSString *)getPrivacyOptionsRequirementStatusString:
                                                                formError.localizedDescription,
                                                          } mutableCopy]];
                                   } else {
-                                    resolve(@"Privacy options form presented successfully.");
+                                    resolve([self getConsentInformation])
```

Comment:
error: expected ';' after expression resolve([self getConsentInformation]).

Reply:
Looks like I actually one ran the Android tests :O Just fixed both.